### ISSUE TITLE
move a11y strings into main strings json

### ISSUE THREAD

**Issue by zepumph** [phetsims/balloons-and-static-electricity#463]
From phetsims/chipper#795. I think that @jessegreenberg should do this, because likely there should be some organizational refactoring done in this repo, in part because there are so many strings here.

**Comment by jbphet** [comment]
This issue was reviewed during today's sound design meeting, and it looks like the a11y strings are still separate from the other strings (they are in `BASEA11yStrings.js`).  This doesn't seem like a blocking issue for the upcoming publication of version 1.5.0, but we thought it best to check.  @jessegreenberg - Should we try to get this done now, or leave it for the next publication from master?

**Comment by zepumph** [comment]
I'll put in a strong preference to doing this sooner rather than later, and not keeping this technical debt lingering.

**Comment by jessegreenberg** [comment]
Agreed this should be done! But this also doesn't need to block taking RC SHAs. I won't be able to get to this until 9/7/21 at the earliest, @jbphet if you would like to take RC SHAs before then I think that is fine.

**Comment by jessegreenberg** [comment]
This is being handled by #604 .
